build/commands/lib/config.js

```diff
@@ -57,6 +57,13 @@ const getNPMConfig = (key, default_value = undefined) => {
   if (!NpmConfig) {
     const list = run(npmCommand, ['config', 'list', '--json', '--userconfig=' + path.join(rootDir, '.npmrc')])
     NpmConfig = JSON.parse(list.stdout.toString())
+    // Merge in config from `.env` file
+    require('dotenv').config({ processEnv: NpmConfig, override: true })
```

Comment:
nit: put `require('dotenv')` at the top of the file and call config here

Reply:
good call, done